server/src/app/auth/auth.service.spec.ts

```diff
@@ -144,8 +148,8 @@ describe('AuthService', () => {
 
       const user: User = new User(address, name);
       const jwt: Token = {
-        expiresIn: 3600,
```

Comment:
i think declaration of these variables can be located below describe('signIn') to see it easily for other people

Reply:
@inDlife It's outdated! No worries :)